### ISSUE TITLE
Custom Fields Tutorials Series

### ISSUE THREAD

**Issue by alann60** [max123kl/J4.docs#8]
Make copy of existing J3 docs and refactor for Joomla 4 Namespace.
Refer to Supporting Info https://github.com/max123kl/J4.docs/blob/main/J4_JDocs/Supporting_Documents/j4x_custom_fields_tutorial_series.md

**Comment by ceford** [comment]
The Subform field does not exist in J3. Should its creation be a separate issue? Perhaps more urgent than updating the J3 versions?

**Comment by alann60** [comment]
I added this more to see how to do it as I'd put it on a Mattermost board.

With 17 docs plus the knock on effects you find in each (i.e. links going to other docs that need updating) I personally would have gone with one task with subtasks. But that was before we started looking at this repo. It also maybe makes sense for someone to do the whole series.

You are right on doing the subforms one first, just to get it out there as it is perhaps the hardest to grasp.

**Comment by max123kl** [comment]
@ceford 
This is the question we need to work on.
We need to develop procedures on how the workflow is easiest to deal with.
My feeling is that it might be a good idea to start a new issue.

@alann60 
The advantage of communicating here on GH is that any reference/link I mention to another thread is automatically registered in both threads.
[See here](https://github.com/max123kl/J4.docs/discussions/6#discussion-4768759)

**Comment by alann60** [comment]
I agree - I've amended this.
